src/main/resources/yamlschema/PrintResponses.yaml

```diff
@@ -0,0 +1,15 @@
+#version: 0.0.3
+title: PrintResponses
```

Comment:
`PrintResponses` ... this is an interesting one @kristerbone ....
Given that it's a type that contains 2 properties which are both arrays of other types ..... how will this get serialized as a flat PSV ? 🤔